### ISSUE TITLE
LOCK: The process cannot access the file because it is being used by another process

### ISSUE THREAD

**Issue by syzer** [workshopper/levelmeup#42]
`levelmeup run example11.js`

`levelmeup verify example11.js`

gives :

```
events.js:85
      throw er; // Unhandled 'error' event
OpenError: IO error: LockFile ~levelmeup_1_4520/LOCK: The process cannot access the file because it is being used by another process.
```

**Comment by syzer** [comment]
forgot file
```js
'use strict';
var level = require('level');
var sub = require('level-sublevel');
var db = sub(level(process.argv[2]));
var _ = require('lodash');

var robots = db.sublevel('robots');
var dinosaurs = db.sublevel('dinosaurs');

[robots, dinosaurs].map(function (lev, i) {
    lev.put('slogan', 1 === i ? 'rawr' : 'beep boop');
});
```

seems that loading lodash creates a problem!

**Comment by martinheidegger** [comment]
This should be fixed in #92
